packages/code-du-travail-frontend/src/outils/IndemniteLicenciement/stepReducer.js

```diff
@@ -53,11 +53,11 @@ export function stepReducer(steps, action) {
     }
     case "add_branche": {
       return steps
-        .filter(step => !/branche_/.test(step.name))
+        .filter(step => !/branche/.test(step.name))
```

Comment:
if you revert the changes of the step name, do not forget that one :)

Reply:
yeeeep